### PR TITLE
Branch ozan#210

### PR DESCRIPTION
**Pull request by OzanYRT**
Corrected a lot of checkstyle errors.

### CODE REVIEW

**Review by bli129 — APPROVED** [review]
The names need work, but this looks good as an initial constant refactor

File: src/main/java/org/bcit/comp2522/project/CollectionManager.java [hunk 1]
```diff
@@ -1,10 +1,11 @@
 package org.bcit.comp2522.project;
 
+import static org.bcit.comp2522.project.ConstantManager.ZERO;
```
Comment (breader1) [inline src/main/java/org/bcit/comp2522/project/CollectionManager.java:3]: This variable is static and should be referenced by the class. We don't want a bunch of imports like this

Example: ConstantManager.ZERO.

**Review by breader1 — CHANGES_REQUESTED** [review]
I'm going to close this PR. There are too many imports when the variables we're referencing are static variables. 

We can talk as a group about a better option tomorrow, but for now, these changes would make the code require more refactoring.

**Comment by breader1** [comment]
Too many imports on static class variables. We'll figure out a better strategy as a group sometimes on Friday.

**Pull request closed without merging**